src/pydvl/influence/general.py

```diff
@@ -2,32 +2,15 @@
 Contains parallelized influence calculation functions for general models.
 """
 from enum import Enum
-from typing import TYPE_CHECKING, Callable, Dict, Optional
 
 import numpy as np
-from scipy.sparse.linalg import LinearOperator
+from numpy.typing import NDArray
 
 from ..utils import maybe_progress
-from .conjugate_gradient import (
-    batched_preconditioned_conjugate_gradient,
-    conjugate_gradient,
-)
-from .frameworks import TorchTwiceDifferentiable
-from .types import MatrixVectorProductInversionAlgorithm, TwiceDifferentiable
+from .frameworks import TensorType, TwiceDifferentiable
+from .inversion_methods import InversionMethod, invert_matrix
 
-try:
-    import torch
-    import torch.nn as nn
-
-    _TORCH_INSTALLED = True
-except ImportError:
-    _TORCH_INSTALLED = False
-
-if TYPE_CHECKING:
-    from numpy.typing import NDArray
-
-
-__all__ = ["compute_influences", "InfluenceType", "InversionMethod"]
+__all__ = ["compute_influences", "InfluenceType", "calculate_influence_factors"]
```

Comment:
Again, consistency: either "compute" or "calculate", pick one everywhere! Value uses "compute", so I'd stick with that